lib/ITip/Broker.php

```diff
@@ -855,7 +855,10 @@ protected function parseEventInfo(VCalendar $calendar = null) {
                 $sequence = $vevent->SEQUENCE->getValue();
             }
             if (isset($vevent->EXDATE)) {
-                $exdate = $vevent->EXDATE->getParts();
+                foreach ($vevent->select('EXDATE') as $val) {
+                    $exdate = array_merge($exdate, $val->getParts());
```

Comment:
Do we consider the `PRIORITY` property here to sort `EXDATE` before merging them? Because order is important when merging.